cmd/kubeadm/app/constants/constants.go

```diff
@@ -123,8 +123,7 @@ const (
 	KubeConfigVolumeName = "kubeconfig"
 
 	// NodeBootstrapTokenAuthGroup specifies which group a Node Bootstrap Token should be authenticated in
-	// TODO: This should be changed in the v1.8 dev cycle to a node-BT-specific group instead of the generic Bootstrap Token group that is used now
-	NodeBootstrapTokenAuthGroup = "system:bootstrappers"
+	NodeBootstrapTokenAuthGroup = "system:bootstrappers:kubeadm:default-node-token"
```

Comment:
What happens if you use a kubeadm 1.8 version to deploy a 1.7 cluster will this work?